src/main/java/org/quiltmc/loader/impl/plugin/SolverErrorHelper.java

```diff
@@ -797,8 +805,12 @@ void report(QuiltPluginManagerImpl manager) {
 			report.append(", which is present!");
 			error.appendReportText(report.toString(), "");
 
+			if (!reason.isEmpty()) {
+				error.appendReportText("Breaking mod's reason: " + reason, "");
+			}
+
 			for (ModLoadOption mod : from) {
-				error.appendReportText("- " + manager.describePath(mod.from()));
+				error.appendReportText("- " + manager.describePath(mod.from()), "");
```

Comment:
Was this additional newline intentional? I'm not sure it's necessary here.
AFAIK it would make the report look like this:
```
BadCraft-A breaks all versions of libnetworkstack, which is present!

Breaking mod's reason: I don't like LibNetworkStack lol

- <mods>/badcraft-1.0.0.jar!/META-INF/jars/badcraft-submod-a.jar

- <mods>/badcraft-1.0.0.jar!/META-INF/jars/badcraft-submod-b.jar

- <mods>/badcraft-1.0.0.jar!/META-INF/jars/badcraft-submod-c.jar

- <mods>/badcraft-1.0.0.jar!/META-INF/jars/badcraft-submod-d.jar
```

Reply:
Oh no it was not